packages/story-editor/src/components/checklist/countContext/test/checkCountContext.js

```diff
@@ -31,7 +33,17 @@ import {
   useRegisterCheck,
 } from '../checkCountContext';
 
+let consoleErrorFn;
+
 describe('ChecklistCategoryProvider', () => {
+  beforeEach(() => {
+    consoleErrorFn = jest.spyOn(console, 'error').mockImplementation(() => {});
```

Comment:
Curious why this was added.

Reply:
The jsdom environment has a "fake" console that catches thrown errors, which it then passes on to Jest. So Jest then displays the actual console error in the test output. The error is of course expected in this test (using `toThrow()`), but we don't need to see it int the log.

Before that, I think the `@wordpress/jest-console` package was silently swallowing this error for some reason. But since I removed that, now in Jest 28 the message was properly surfaced.

So it's a combination of things that led to this.